vividus-plugin-mobile-app/src/main/resources/properties/profile/mobile_app/profile.properties

```diff
@@ -2,3 +2,5 @@ spring.profiles.active=mobile_app
 
 mobile-app=true
 mobile-application.session-scope=story
+mobile-application.swipe.stabilization-duration=PT1S
+mobile-application.swipe.limit=10
```

Comment:
How much in % takes one swipe?

Reply:
depends on swipe duration :)

Reply:
My thoughts here: this is to small value? How many screens it will be able to swipe until the limit reached?

Reply:
I would say that 10 is good for most apps I previously meet